index.js

```diff
@@ -202,6 +323,13 @@ function loader (registryOrVersion) {
       if (registry.supportFeature('spawnEggsHaveSpawnedEntityInName')) {
         return this.name.replace('_spawn_egg', '')
       }
+      if (registry.type === 'bedrock') {
+        // TODO: add to features.json
+        // According to the wiki, the different metadata values were split
+        // into their own IDs in 1.16.100
+        // https://minecraft.fandom.com/wiki/Spawn_Egg#History
```

Comment:
Since spawn eggs don't change much, indeed if it's necessary a manual map of item name to entity name may make sense

Reply:
I don't think that's necessary, as far as I can tell, the mob's name is in the item's name, and in older versions it uses the entity's internalId in metadata
metadata is still supported in the current version (in commands such as /give) but I think the spawn egg names are used in protocol, I'll look into that